app/src/main/res/values-v21/styles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+  -->
+
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+    <style name="Transpo" parent="Theme.MaterialComponents.DayNight.NoActionBar" tools:targetApi="27">
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
+        <item name="toolbarStyle">@style/Widget.Transpo.Toolbar</item>
+        <item name="android:navigationBarColor">?colorSurface</item>
+        <item name="android:statusBarColor">@color/white</item>
```

Comment:
Consider using a general attribute instead just "white". An attribute could be reused and applying the night theme will be easier this way.